src/app/radio/page.tsx

```
@@ -0,0 +1,167 @@
+          <button
+            className="text-gray-50 absolute top-4 right-8 cursor-pointer"
+            onClick={() => {
+              navigator.clipboard.writeText(propTypes);
+            }}
+            title="Copy props to clipboard"
+          >
+            <Copy />
+          </button>
+          <p
+            className="flex flex-col mx-auto bg-gray-100 p-2 w-full h-full overflow-auto"
+            dangerouslySetInnerHTML={{ __html: sanitizeHtml(propType) }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)